src/rantanplan/core.py

```diff
@@ -740,5 +740,10 @@ def generate_liaison_positions(syllables, liaison):
 
 
 def has_single_liaisons(liaisons):
-    print('\n----------------------------', '\n', liaisons, '\n----------------------------')
+    """Returns all combinations that have no consecutive liaisons
```

Comment:
It returns `True` or `False`. So maybe something on the line of `Checks whether liaisons (a list of 1's and 0's) has consecutive liaisons (1's) or not`.